### ISSUE TITLE
request: show extension status - both if it works and if it doesn't/isn't supposed to

### ISSUE THREAD

**Issue by sesam** [privacypass/challenge-bypass-extension#43]
Requests:
 - explain clearly visibly in {extension, README.md, website} when the extension is not supposed to work (cc #14)
 - Add an after extension install screen that shows a short explanation how it is supposed to work.
 - during usage: show some life signs - maybe like "toast" when new passes are acquired or not. Maybe console.log these events.

I dug through the web, some blog posts, README.md, and then have been following this github project for months. I still don't know if there are users for whom this actually works. I think this extension is supposed to work for websites that:
 - are served via Cloudflare
 - but only if you have got "passes". Do these passes (or count of used/available) show anywhere without prying into the extension internals (like background.html)?

**Comment by sesam** [comment]
I just retested, and to be clear: right after install, I get a gray icon in the toolbar and 0 passes.
Visiting https://captcha.website/ I see no effect, and still have 0 passes. In the JS console nothing is shown.

As a user, the shortest way I can explain this to myself, is that something isn't working.

**Comment by alxdavids** [comment]
From README.md:
```
- Go to a web page supporting Privacy Pass where internet challenges are displayed (e.g. https://captcha.website)
```
```
- Solve CAPTCHA and check that some passes are stored in the extension now

    captcha.website cannot be bypassed (this is only for gaining passes)

```

**Comment by sesam** [comment]
tl;dr?

repro:
1. install extension
2. visit captcha.website
3. nothing happens

(this close made unhappy)

PRs welcome?

**Comment by oxwivi** [comment]
From what I understand, you can get Cloudflare blocks to work on mobile if you get passes from captcha.website. However, captcha.website itself is protected by Cloudflare, so I can't even access it after installing the add-on.

What kind of ridiculousness is this? Why does Cloudflare checks stop working after installing the add-on anyway?

**Comment by oxwivi** [comment]
I also don't understand why this issue is closed, because I've seen nowhere mentioned that the add-on is not supposed to work with Cloudflare until I stumbled upon this issue.
